tests/unit_tests/db_engine_specs/test_trino.py

```diff
@@ -81,7 +84,7 @@ def _assert_columns_equal(actual_cols, expected_cols) -> None:
 @pytest.mark.parametrize(
     "extra,expected",
     [
-        ({}, {"engine_params": {"connect_args": {"source": USER_AGENT}}}),
+        ({}, {"engine_params": {"connect_args": {"source": "Apache Superset"}}}),
```

Comment:
bycatch: I think it's usually better to assert literal expected values (e.g. in case the original constant becomes corrupted by mistake)